6thShopProject2025/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -53,6 +64,21 @@ private void configureBindings() {
       this.joyConRight = mechController.getRightY(); // Right Y-axis for PS5 controller
       tankSubsystem.setMotors(joyConLeft, joyConRight);
     }, tankSubsystem));
+
+    mechController.square().onTrue(new InstantCommand(() -> {
+      if (beltSwitch.get()) {
+        beltSubsystem.breakRoller(true);
+        ;
+      } else {
+        beltSubsystem.setBelt(0.2);
+      }
+    }));
+    mechController.circle().onTrue(new InstantCommand(() -> {
+      beltSubsystem.breakRoller(false);
+      beltSubsystem.setBelt(0.2);
+
+    }));
+
```

Comment:
you will want an option to run the belt backwards in case of unforseen circumstances.